src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -121,4 +122,19 @@ public static Set<Tag> parseTags(Collection<String> tags) throws ParseException
         }
         return tagSet;
     }
+
+    /**
+     * Parses a {@code String phone} into a {@code Phone}.
+     * Leading and trailing whitespaces will be trimmed.
```

Comment:
I think for the javadocs for this part, you are referring to "Parses a {@code String priority} into a {@code Priority}"

Reply:
Good Catch! I will resolve it.